web/l10n/en.json

```diff
@@ -84,6 +84,11 @@
     "deviceCommand": "Command",
     "deviceFollow": "Follow",
     "deviceTotalDistance": "Total Distance",
+    "deviceStatus": "Status",
+    "deviceSearchFor": "Search for...",
```

Comment:
Unfortunately it is not empty
![image](https://cloud.githubusercontent.com/assets/5688080/24233579/fdc4739a-0fb4-11e7-9d5f-2dd91bf338fd.png)